docs/1___overview.adoc

```diff
@@ -194,6 +198,7 @@ It is possible to distribute an FMU without knowing the target processor.
 This allows an FMU to run on a PC, a Hardware-in-the-Loop simulation platform or as part of the controller software of an ECU.
 Keeping the FMU independent of the target processor increases the usability of the FMU.
 To be processor independent, the FMU must include its C (or C++) sources.
+To be maximally portable, FMUs must reduce their dependency on OS services and use these only through standard library calls.
```

Comment:
Add? "...any dependency ... documentation/externalDependencies.{txt|html} .... "

Reply:
Sorry, I am not sure what you mean... the need for documentation was covered above. "their dependency" means "all". What is missing?